core/src/com/unciv/models/metadata/GameSettings.kt

```diff
@@ -45,6 +45,8 @@ class GameSettings {
 
     var showMinimap: Boolean = true
     var minimapSize: Int = 6    // default corresponds to 15% screen space
+    var unitIconOpacity = 1f // default corresponds to fully opaque
+    var unitIconSize = 25f // default corresponds to maximum size
```

Comment:
This might be better as a percentage slider that has a default of 100% (corresponding to `25f`). The average user probably doesn't care about the size in floats but knows that 200% means double the size and 50% halves it.

Reply:
Good point. I can set the max/default value of 25f to 100% and the min to 40% (10 out of 25).

Reply:
Yep, I updated it to go from 40% to 100% in 5% increments.